client/app/bundles/course/material/folders/pages/ErrorRetrievingFolderPage.tsx

```diff
@@ -0,0 +1,53 @@
+import { defineMessages } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { Cancel, FolderOutlined } from '@mui/icons-material';
+
+import Link from 'lib/components/core/Link';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import MaterialStatusPage from '../../components/MaterialStatusPage';
+
+const translations = defineMessages({
+  problemRetrievingFolder: {
+    id: 'course.material.folders.ErrorRetrievingFolderPage.problemRetrievingFolder',
+    defaultMessage: 'Problem retrieving folder',
+  },
+  problemRetrievingFolderDescription: {
+    id: 'course.material.folders.ErrorRetrievingFolderPage.problemRetrievingFolderDescription',
+    defaultMessage:
+      "Either it no longer exists, you don't have the permission to access it, or something unexpected happened when we were trying to retrieve it.",
+  },
+  goToTheWorkbin: {
+    id: 'course.material.folders.ErrorRetrievingFolderPage.goToTheWorkbin',
+    defaultMessage: 'Go to the Workbin',
```

Comment:
Workbin refers to the customised `Material` title. Since our default naming for `Material` title is not `Workbin`, please change this to the more generic one

I suggest `Go to the Main Folder` is more general to be used

Reply:
done